requirements/py35.txt

```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    ./requirements-compile.sh
+#    ./requirements/compile.sh ./setup.py requirements/requirements.in
```

Comment:
bah this doesn't work yet due to https://github.com/jazzband/pip-tools/issues/908

Reply:
Just put `pkgutil-resolve-name` in the requirements.in file and revert attempts to change the way requirements are compiled (I'd like to keep it uniform across my projects)

Reply:
the ./requirements-compile.sh script doesn't exist and is missing the path to the requirements.in file

Reply:
please rebase on master after #202 rewrote the script to Python to avoid those working directory problems